src/app/components/TokenCircle.tsx

```diff
@@ -33,7 +33,7 @@ export const TokenCircle = ({
         return (
           <Avatar
             key={index}
-            className="absolute w-16 h-16 rounded-full"
+            className="absolute h-16 w-16 rounded-full"
```

Comment:
we can use `size-16` instead of `h-16 w-16`

Reply:
Please update the change in other places as well.

Reply:
I found and fixed them all 😎